compiler/rustc_builtin_macros/src/log_syntax.rs

```diff
@@ -1,14 +1,14 @@
 use rustc_ast::tokenstream::TokenStream;
 use rustc_ast_pretty::pprust;
-use rustc_expand::base;
+use rustc_expand::base::{self, ExpandResult};
 
 pub fn expand_log_syntax<'cx>(
     _cx: &'cx mut base::ExtCtxt<'_>,
     sp: rustc_span::Span,
     tts: TokenStream,
-) -> Box<dyn base::MacResult + 'cx> {
+) -> base::MacroExpanderResult<'cx> {
```

Comment:
```suggestion
) -> MacroExpanderResult<'cx> {
```
Could you import `ExpandResult` and `MacroExpanderResult` consistently in all places?